txstratum/cli.py

```diff
@@ -23,6 +23,7 @@ def create_parser() -> ArgumentParser:
     parser.add_argument('--stratum-port', help='Port of Stratum server', type=int, default=8000)
     parser.add_argument('--api-port', help='Port of TxMining API server', type=int, default=8080)
     parser.add_argument('--log-config', help='Config file for logging', default='log.conf')
+    parser.add_argument('--json-logs', help='Enabled logging in json', default=False, action='store_true')
```

Comment:
From what I understood of the implementation, this will make `--log-config` be completely ignored. I think we should either not allow both arguments to be set or at the very least log a warning/error when both are set (noting that `--log-config` will be ignored)

Reply:
I'll improve this.

Reply:
> I think we should either not allow both arguments to be set 

Seems like the problem with this option is that `--log-config` has a default value. So it is always set.

I could check if it is set and different from default, and forbid this case. And log the warning/error every time `--json-logs` is set and `--log-config` is the default value.

What do you think?

Reply:
Done in https://github.com/HathorNetwork/tx-mining-service/pull/45/commits/f36fe89c1272e2568270bcf3d67aba6f0bd193fa